Week3/functionVehicle.js

```diff
@@ -0,0 +1,19 @@
+// 6. Create a function called vehicle, like before, but takes another parameter called age, so that vehicle("blue", 1, 5) prints 'a blue used car'
+
+function vehicle(color, code, age) {
+  if (age >= 1 && code === 1) {
```

Comment:
Looks good, you can do this easier by splitting the `if-statements`.

You can make one to determine the `age`. So you could do this:

```
if (age >= 1) {
 ageText = `new`;
} else {
 ageText = `used`
}
```

And the  same for the rest, if you understand what I mean. But this looks good anyways, but it's just a suggestion

Reply:
I changed it, its better now, thanks.